core/templates/services/context.service.spec.ts

```diff
@@ -244,14 +305,62 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that skill can refer to skills',
```

Comment:
'skill can refer to skills' seems wrong....

Reply:
Also, isn't this a duplicate test? See Line 219.

Reply:
Removed. Done!